lib/server-builder.js

```diff
@@ -61,15 +74,31 @@ class ServerBuilder {
 
   getCommand () {
     const cmd = system.isWindows() ? 'gradlew.bat' : './gradlew';
+    let buildProperty = function (key, value) {
+      return value ? `-P${key}=${value}` : null;
+    };
     let args = VERSION_KEYS
       .filter(key => key !== GRADLE_VERSION_KEY)
       .map(key => {
         const serverVersion = this.serverVersions[key];
         const gradleProperty = `appium${key.charAt(0).toUpperCase()}${key.slice(1)}`;
-        return serverVersion ? `-P${gradleProperty}=${serverVersion}` : null;
+        return buildProperty(gradleProperty, serverVersion);
       })
       .filter(Boolean);
 
+    if (this.signingConfig) {
+      args.push(
```

Comment:
```
args.push(...(
['zipAlign', 'keystoreFile', 'keystorePassword', 'keyAlias', 'keyPassword']
  .map((k) => [`appium${_.capitalize(key)}`, this.signingConfig[key]])
  .map(([k, v]) => buildProperty(k, v))
));
```

Reply:
or simply that (if we know we only want the items of `signingConfig`):

```
args.push(...(
  _.keys(this.signingConfig)
    .map((key) => [`appium${_.capitalize(key)}`, this.signingConfig[key]])
    .map(([k, v]) => buildProperty(k, v))
));
```

Reply:
also then there will be no need for to extract the `buildProperty` helper as it could be simply put to the lambda in `map`